test/OWNERS

```diff
@@ -0,0 +1,21 @@
+reviewers:
+  - deads2k
+  - p0lyn0mial
+  - tkashem
+  - benluddy
+  - sanchezl
+  - dgrisonnet
+  - wangke19
+  - gangwgr
+  - xingxingxia
+approvers:
+  - deads2k
+  - p0lyn0mial
+  - tkashem
+  - benluddy
+  - sanchezl
+  - dgrisonnet
+  - wangke19
+  - gangwgr
+  - xingxingxia
+component: openshift-apiserver
```

Comment:
Do we need to add a new OWNERS file now? 

I’d prefer to clarify a few details first, for example, which framework we’re using, what we’re doing with the existing tests, and when new tests should be added to the test binary.